packages/airnode-node/config/config.example.json

```diff
@@ -33,7 +33,7 @@
     },
     "logFormat": "plain",
     "logLevel": "INFO",
-    "nodeVersion": "0.5.0",
+    "nodeVersion": "0.4.0",
```

Comment:
Maybe we need some kind of script we can run to update all of these values. I assume you're doing this manually now?

Reply:
Yes, but it will be pretty easy to do so - after a few minor changes you will be able to just globally find and replace them (e.g in IDE) - doing it with script is likely an overkill.

Reply:
I would definitely welcome a script for that. You need to update:
* All the examples in airnode-examples
* Actual code in examples
* Example in airnode-deployer
* Example in airnode-node
* airnode-validator multiple places (guess that won't be an issue anymore)

I did it few times and I almost always missed at least one place :smile:

Reply:
Makes sense. I'll add a TODO to github description.